myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/event/handlers/ResourceOffersEventHandler.java

```diff
@@ -116,8 +114,21 @@ public void onEvent(ResourceOffersEvent event, long sequence,
                         driver.declineOffer(offer.getId());
                     }
                 }
-            } else {
-                offerLifecycleMgr.addOffers(offers);
+            }
+
+            for (Offer offer : offers) {
+              if (SchedulerUtils.isEligibleForFineGrainedScaling(offer.getHostname(), schedulerState)) {
```

Comment:
Fixed in https://github.com/smarella/myriad/commit/ef328d15ffc7fd378fe7ce8417b665d462cbfa72